Makefile

```diff
@@ -128,3 +128,26 @@ setup-env:
 	venv/bin/pip install --upgrade -r build_calicoctl/requirements.txt
 	@echo "run\n. venv/bin/activate"
 
+
+run-etcd-secure:
+	@-docker rm -f calico-etcd
+	docker run --detach \
+	--net=host \
+	-v `pwd`/certs:/etc/calico/certs \
+	--name calico-etcd quay.io/coreos/etcd:v2.0.11 \
+	--cert-file "/etc/calico/certs/test-cert.crt" \
```

Comment:
would these test-certs be better named server-cert to distinguish from the client-cert?